src/js/MarketData.jsx

```diff
@@ -0,0 +1,202 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import axios from 'axios';
+import moment from 'moment';
+import styled from 'styled-components';
+import {
+  setDuration,
+  setCoin,
+  setHoverLoc,
+  setActivePoint,
+  setFetchingData
+} from './actionCreators';
+import getData from './asyncActions';
+import LineChart from './LineChart';
+import ToolTip from './ToolTip';
+import InfoBox from './InfoBox';
+import Spinner from './Spinner';
+
+const coinCode = {
+  Bitcoin: 'BTC',
+  Ethereum: 'ETH',
+  Litecoin: 'LTC',
+  BTC: 'Bitcoin',
+  ETH: 'Ethereum',
+  LTC: 'Litecoin'
+};
+
+class MarketData extends Component {
+  componentDidMount() {
+    this.props.handleGetData(this.props);
+  }
+
+  // getData1(url) {
+
+  render() {
+    const Wrapper = styled.h2`
+      color: #fc4a1a;
+    `;
+
+    const { duration, coin } = this.props;
+    const style = {
+      'text-decoration': 'none',
+      border: 'none'
+    };
+    const bold = {
+      'font-weight': 'bold',
+      'text-decoration': 'none'
+    };
+    const durations = ['1H |', ' 1D |', ' 1W |', ' 1M |', ' 1Y'];
+    return (
```

Comment:
Not necessary but it would be nice to destructure your props up here, so you don't have to keep writing `this.props.whatever` throughout the render method